src/main/java/frc/robot/Robot.java

```diff
@@ -119,45 +119,42 @@ public void teleopInit() {
     // });
     // }
 
-    // public Command goodFlip() {
-    // CommandChain retValue = new CommandChain("Good Flip");
+    public Command goodFlip() {
+        CommandChain retValue = new CommandChain("Good Flip");
 
-    // retValue.then(lift.goToAtLeast(LiftSubsystem.Heights.kLiftFlipHeight)).then(maflipulator.crappyFlip());
-    // return retValue;
-    // }
+        retValue.then(lift.goToAtLeast(LiftSubsystem.Heights.kLiftFlipHeight)).then(maflipulator.crappyFlip());
+        return retValue;
+    }
 
-    // public Command stowAndGo() {
-    // CommandChain retValue = new CommandChain("Stow it and go!");
-    // retValue.then(lift.goToHeight(Heights.kFloorLoad),
-    // maflipulator.stowAndGoPosition());
-    // return retValue;
-    // }
+    public Command stowAndGo() {
+        CommandChain retValue = new CommandChain("Stow it and go!");
+        retValue.then(lift.goToHeight(Heights.kFloorLoad), maflipulator.stowAndGoPosition());
+        return retValue;
+    }
 
```

Comment:
Let's add another command "goToLoadingStation"... or with the dumb old convention, "levelZero"???

Reply:
Done for all hatch and cargo heights